test/units/testsuite-23.ExecStopPost.sh

```diff
@@ -100,7 +102,3 @@ test -f /run/idle1
 test -f /run/idle2
 
 systemd-analyze log-level info
```

Comment:
Please drop `systemd-analyze log-level` or `systemctl log-level` from subtests, but call it from the main script. The subtest scripts are now running in parallel, hence, we may not get expected logs.

Reply:
The subtests are still serialized - the async shenanigans are there just to make traps work, but we still wait for each subtest to finish before running the next one.

Reply:
Ah, indeed. I overlooked the comments in the main script. Interesting.